tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -575,6 +583,7 @@ private ByteBuffer getTokens() throws IOException, URISyntaxException, YarnExcep
                 + "other namenode: " + namenodeUri);
           }
           cred.addToken(otherFSToken.getService(), otherFSToken);
+          LOG.info("Fetched HDFS token for " + nnUri);
```

Comment:
Suggest also adding a matching `"Fetching HDFS token for " + nnUri"` LOG statement. This lets us measure token fetching delay.

Reply:
agreed. Also would be nice to have logging for default FS delegation token as well